pkg/operator/ceph/object/user/controller.go

```diff
@@ -403,6 +403,15 @@ func generateUserConfig(user *cephv1.CephObjectStoreUser) admin.User {
 		userConfig.MaxBuckets = user.Spec.Quotas.MaxBuckets
 	}
 
+	if user.Spec.AccessKey != "" && user.Spec.SecretKey != "" {
```

Comment:
Please add a unit test for this case. You can follow other examples in controller_test.go that call `generateUserConfig()`.

Reply:
done